src/juju/actions.js

```diff
@@ -71,23 +70,6 @@ export function updateModelInfo(modelInfo) {
 
 // Thunks
 
-/**
-  Fetches the model list from the supplied Juju controller. Requires that the
-  user is logged in to dispatch the retrieved data from listModels.
-  @param {Object} conn The controller connection.
-  @returns {Object} models The list of model objects under the key `userModels`.
-*/
-export function fetchModelList(conn) {
```

Comment:
I find myself removing thunks more and more, I'm wondering if we shouldn't revisit another architecture...